### DISCUSSION TITLE
Add support for Tabler UI

### DISCUSSION THREAD

**Discussion by peter-mw** [filamentphp/filament#6138] [Ideas]
Hello,

Thanks for the beautiful framework!

For people like us who are bound to Bootstrap because of legacy code, can you add support for Tabler UI as an alternative to Tailwind? 

The Tabler UI theme will allow people who are bound to Bootstrap to use Filament without problems. 

And it still keeps the `T` in `TALL` stack.

https://github.com/tabler/tabler

https://preview.tabler.io/

**Comment by danharrin** [comment]
I appreciate the use case, but no. It would make Filament unmaintainable, and none of the core team have any interest in Bootstrap and we would never use it ourselves. Maintaining heavy code that you'd never use yourself is a dangerous path for any developer.
